master/buildbot/master.py

```diff
@@ -347,6 +347,10 @@ def doReconfig(self):
         log.msg("beginning configuration update")
         changes_made = False
         failed = False
+
+        yield self.data.updates.masterStopped(
```

Comment:
why? we dont stop the master during the reconfig, that is the whole purpose of reconfig.